imus:
  gimbal_imu:
    frame_id: gimbal_imu
    orientation_covariance_diagonal: [ 0.0012, 0.0012, 0.0012 ]
    angular_velocity_covariance: [ 0.0004, 0.0004, 0.0004 ]
    linear_acceleration_covariance: [ 0.01, 0.01, 0.01 ]
  base_imu:
    frame_id: base_imu
    orientation_covariance_diagonal: [ 0.0012, 0.0012, 0.0012 ]
    angular_velocity_covariance: [ 0.0004, 0.0004, 0.0004 ]
    linear_acceleration_covariance: [ 0.01, 0.01, 0.01 ]
